### ISSUE TITLE
[APP SUBMITTED]: [Errno 2] No such file or directory: '/volume1/video/Sous-titres/MythBusters'

### ISSUE THREAD

**Issue by boyahr** [SiCKRAGETV/sickrage-issues#2065]
### INFO
Python Version: **2.7.10 (default, Jun  3 2015, 17:38:42) [GCC 4.6.4]**
Operating System: **Linux-3.2.40-armv7l-with-glibc2.4**
Locale: unknown
Branch: **master**
Commit: SiCKRAGETV/SickRage@25649f25806373d1a70608add3c00d1d371b71bf
Link to Log: https://gist.github.com/b0e047fb6ee600783649
### ERROR
```
FINDSUBTITLES :: Error occurred when downloading subtitles: Traceback (most recent call last):
  File "/volume1/@appstore/sickrage/var/SickRage/sickbeard/tv.py", line 1516, in downloadSubtitles
    helpers.chmodAsParent(subs_new_path)
  File "/volume1/@appstore/sickrage/var/SickRage/sickbeard/helpers.py", line 597, in chmodAsParent
    childPathStat = ek.ek(os.stat, childPath)
  File "/volume1/@appstore/sickrage/var/SickRage/sickbeard/encodingKludge.py", line 64, in ek
    result = func(*[ss(x) if isinstance(x, (str, unicode)) else x for x in args], **kwargs)
OSError: [Errno 2] No such file or directory: '/volume1/video/Sous-titres/MythBusters'

```
---
_STAFF NOTIFIED_: @SiCKRAGETV/owners @SiCKRAGETV/moderators

**Comment by miigotu** [comment]
Is 'Sous-titres' the right path or is there missing accents?

**Comment by boyahr** [comment]
It's the right path, the problem is that Sickrage shouldn't look deeper than "/Sous-titres", because it is the subtitle directory.  Oddly, for MythBusters, it started to look for "/Sous-titres/MythBusters" which doesn't exist (and shouldn't), I have no idea why...

**Comment by miigotu** [comment]
It is supposed to put the subtitles where the shows are. Problem here is that you have post process dir inside your show dir.

`the problem is that Sickrage shouldn't look deeper than "/Sous-titres"`

It absolutely should, if that is your set post process dir.

Your settings are wrong.
Have torrent client download in one folder, then move completed to a "completed downloads" folder (not inside your tv shows folder). It can be called whatever you want
Set post process to this "completed downloads" folder
in subtitles settings, erase the path in the box.

**Comment by miigotu** [comment]
There is a planned feature to prevent a user from even being able to set a post process dir inside the tv shows dir. post processor isn't supposed to operate on source files inside the tv shows dir, only as a destination.

**Comment by miigotu** [comment]
Just empty the subtitles dir box first, and try again.

![image](https://cloud.githubusercontent.com/assets/390379/8637700/4bc6f2f6-2854-11e5-80ca-6235a65067d7.png)

**Comment by boyahr** [comment]
I'm using the DS Downloader of a Synology to download torrents, but there's no option (to my knowledge) for it to have an incomplete and a complete downloads directory.  However, that was not the point.

There wasn't any post process inside the tv shows dir, DS Download usually drop its files into a sibling folder, and post process acts on it.  I also made another sibling directory for subtitles ("Sous-titres"), because of a raspberry pi I use with Kodi (which offers to download subtitles into a directory).

Anyway, I disabled it as you said and I don't have that error anymore, but I'm now wondering how Kodi will react to it...
